src/main/java/com/ent/staySafe/entity/StaySafe.java

```diff
@@ -28,19 +28,19 @@
 	private String roomNumber;
 	
 	@Column(name="checkin_Date")
-	private String checkinDate;
+	private String checkInDate;
 	
 	@Column(name="checkout_Date")
-	private String checkoutDate;
+	private String checkOutDate;
 	
```

Comment:
Oracle Java variable naming conventions, "Except for variables, all instance, class, and class constants are in mixed case with a lowercase first letter. Internal words start with capital letters. Variable names should not start with underscore _ or dollar sign $ characters, even though both are allowed."
I capitalized this variable name to adhere to these conventions

https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html